install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -40,6 +40,14 @@ spec:
       for: 10m
       labels:
         severity: critical
+    - alert: ClusterVersionOperatorNotRetrieving
```

Comment:
nit: The CVO retrieves a few things (including Cincy graph data, release signatures, and Kube API objects).  Can we call this `ClusterVersionOperatorUpdatesStale` or `ClusterVersionOperatorAvailableUpdatesStale` or some such?

Reply:
From doc https://github.com/wking/cluster-version-operator/blob/master/docs/user/status.md:

"When RetrievedUpdates is False, ... In all cases, the impact is that the cluster will not be able to retrieve recommended updates"

So how about ClusterVersionOperatorCannotRetrieveUpdates?

Reply:
sounds great